cmd/honeyelb/main.go

```diff
@@ -76,8 +77,27 @@ Your write key is available at https://ui.honeycomb.io/account`)
 				}
 			}
 
-			// Use this one publisher instance for all ObjectDownloadParsers.
-			stater := state.NewFileStater(opt.StateDir, logbucket.AWSElasticLoadBalancing)
+			var stater state.Stater
+
+			if opt.HighAvail {
+				svc := dynamodb.New(sess)
+				input := &dynamodb.DescribeTableInput{
+					TableName: aws.String("HoneyELBAccessLogBuckets"),
```

Comment:
Could we move the name of the bucket to a `const` to avoid having the same literal string in multiple places (to help avoid typos)